src/datasource.ts

```diff
@@ -390,7 +418,11 @@ export default class CogniteDatasource {
           ids += Object.keys(idsObj).length;
         }
         if (ids === 0) ids = 1; // will fail anyways, just show the api error message
-        queryReq.limit = Math.floor(100_000 / ids);
+
+        // check if any aggregates are being used
+        const usesAggregations = queryList.some(item => item.aliases.length > 0);
+
+        queryReq.limit = Math.floor((usesAggregations ? 10_000 : 100_000) / ids);
```

Comment:
This `Math.floor` calculations is starting to appear multiple places now, we can consider creating a `calculateLimit` function or something.

Reply:
This only shows up in two places, and are right next to each other - no need right now to make such a function.